From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 4 Apr 2018 11:55:52 +0100
Subject: txt2man options

Pass appropriate options to txt2man to get a proper .TH line.
These are -s 1 for the section,
we skip -d xxx for the date, which will default to the
debian/changelog timestamp during a debian package build, or otherwise
to the current date.
Then -r origin-aka-project-name-and-version -v volume-aka-section-name.
---
 CMake/exec2man.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/exec2man.sh b/CMake/exec2man.sh
index b0807fc..8cf34e9 100755
--- a/CMake/exec2man.sh
+++ b/CMake/exec2man.sh
@@ -75,6 +75,6 @@ synopsis="$name [-h] [-v] $reqoptions $options";
   sed 's/  / /g' | \
   awk '/NAME/,/REQUIRED OPTIONS/ { print; } /ADDITIONAL INFORMATION/,0 { print; } /REQUIRED OPTIONS/,/ADDITIONAL INFORMATION/ { if (!/REQUIRED_OPTIONS/ && !/OPTIONS/ && !/ADDITIONAL INFORMATION/) { if (/ --/) { printf "\n" } sub(/^[ ]*/, ""); sub(/ [ ]*/, " "); printf "%s ", $0; } else { if (!/REQUIRED OPTIONS/ && !/ADDITIONAL INFORMATION/) { print "\n"$0; } } }' | \
   sed 's/  ADDITIONAL INFORMATION/\n\nADDITIONAL INFORMATION/' | \
-  txt2man -P mlpack -t "$name" -d 1 | \
+  txt2man -t "$name" -s 1 -r "MLPACK Utilities" -v "User Commands" | \
   sed "s/^'/\\\\(aq/" > "$output"
 
